recipes/recipe/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "openmolcas" %}
```

Comment:
if you using `name` and/or `build` only once, it's usually more readable to avoid the jinja variables, and can make the bot work more smoothly